gtk/docs.md

```diff
@@ -1 +1,362 @@
+
+An example of these properties in UI definitions:
+
+```xml
+<object class="GtkBox">
+  <child>
+    <object class="GtkEntry" id="entry1"/>
+    <packing>
+      <property name="pack-type">start</property>
+    </packing>
+  </child>
+  <child>
+    <object class="GtkEntry" id="entry2"/>
+  </child>
+  <focus-chain>
+    <widget name="entry1"/>
+    <widget name="entry2"/>
+  </focus-chain>
+</object>
+```
+
+# Implements
+
+[`ContainerExt`](trait.ContainerExt.html), [`WidgetExt`](trait.WidgetExt.html)
+<!-- trait ContainerExt -->
+Trait containing all `Container` methods.
+
+# Implementors
+
+[`Bin`](struct.Bin.html), [`Box`](struct.Box.html), [`Container`](struct.Container.html), [`Fixed`](struct.Fixed.html), [`FlowBox`](struct.FlowBox.html), [`Grid`](struct.Grid.html), [`HeaderBar`](struct.HeaderBar.html), [`Layout`](struct.Layout.html), [`ListBox`](struct.ListBox.html), [`MenuShell`](struct.MenuShell.html), [`Notebook`](struct.Notebook.html), [`Paned`](struct.Paned.html), [`ToolItemGroup`](struct.ToolItemGroup.html), [`ToolPalette`](struct.ToolPalette.html), [`Toolbar`](struct.Toolbar.html), [`TreeView`](struct.TreeView.html)
+<!-- trait ContainerExt::fn child_notify -->
+Emits a `Widget::child-notify` signal for the
+[child property](struct.Container.html#child-properties)
+`child_property` on widget.
+
+This is an analogue of `gobject::Object::notify` for child properties.
+
+Also see `Widget::child_notify`.
+## `child`
+the child widget
+## `child_property`
+the name of a child property installed on
+ the class of `self`
+<!-- impl Expander::fn get_use_markup -->
+Returns whether the label’s text is interpreted as marked up with
+the [Pango text markup language](https://developer.gnome.org/pygtk/stable/pango-markup-language.html).
+See `Expander::set_use_markup`.
+
+# Returns
+
+`true` if the label’s text will be parsed for markup
+<!-- impl Expander::fn set_use_markup -->
+Sets whether the text of the label contains markup in
+[Pango’s text markup language](https://developer.gnome.org/pygtk/stable/pango-markup-language.html).
+See `Label::set_markup`.
+## `use_markup`
+`true` if the label’s text should be parsed for markup
+<!-- struct Label -->
+The `Label` widget displays a small amount of text. As the name
+implies, most labels are used to label another widget such as a
+`Button`, a `MenuItem`, or a `ComboBox`.
+
+# `Label` as `Buildable`
+
+The `Label` implementation of the `Buildable` interface supports a
+custom `<attributes>` element, which supports any number of `<attribute>`
+elements. The `<attribute>` element has attributes named “name“, “value“,
+“start“ and “end“ and allows you to specify `pango::Attribute` values for
+this label.
+
+An example of a UI definition fragment specifying Pango attributes:
+
+```C
+  // Pressing Alt+H will activate this button
+  button = gtk_button_new ();
+  label = gtk_label_new_with_mnemonic ("_Hello");
+  gtk_container_add (GTK_CONTAINER (button), label);
+```
```

Comment:
I'd like the examples to be translated to Rust and pass compile-doctests (i.e. marked with `no_run`) where possible.